File: Python Word examples/tables/update_table_format.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.models.table_format import TableFormat
from spirecloudword.models.color import Color
from spirecloudword.models.borders import Borders
from spirecloudword.models.border import Border
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

name = "Template.docx"
index = 0
node_path = "sections/0"
properties = TableFormat()
properties._bidi = True
border = Border()
border._border_type = "ThinThickThinMediumGap"
border._color = Color(255, 25, 1)
border._line_width = 10
properties._borders = Borders(border)
inputFolder = "input"
destFilePath = "output/Tables/update_table_format.docx"
result = api.update_table_format(name, node_path, index, properties, folder=inputFolder, dest_file_path=destFilePath)

File: PHP Word Examples/TablesApiDemo/Table/updateTableFormat.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\TablesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;
use Spire\Cloud\Word\Sdk\Client\Model\Border;
use Spire\Cloud\Word\Sdk\Client\Model\Borders;
use Spire\Cloud\Word\Sdk\Client\Model\Color;
use Spire\Cloud\Word\Sdk\Client\Model\TableFormat;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TablesApi($configuration);

$fileName = "UpdateTableFormat_1_Temp2.docx";
$node_path = 'Section/0/Body/0';
$index = 0;
$properties = new TableFormat(array(
    'bidi'=>true,
    'borders'=>new Borders(array(
        'top_border'=>new Border(array(
            'border_type'=>Border::BORDER_TYPE_DOT,
            'color'=>new Color(array('red' => 255, 'green' => 25, 'blue' => 1)),
            'line_width'=>3
        ))
    ))
));
$folder = 'input';
$storage = null;
$password = null;
$dest_file_path = "Tables/UpdateTableFormat_1.docx";
$apiInstance->updateTableFormat($fileName, $node_path,$index,$properties,$dest_file_path,$folder,$storage,$password);